riptide-spring-boot-autoconfigure/pom.xml

```diff
@@ -261,7 +261,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.0</version>
+                <version>2.22.2</version>
```

Comment:
Looking here, why do we specify plugin version in child pom? I though it will be picked up from parent plugin management section